changelogs/fragments/5502-xfconf-facts-deprecation.yml

```
@@ -0,0 +1,7 @@
+removed_features:
+  - >
+    xfconf - generating facts was deprecated in the commit
+    https://github.com/ansible-collections/community.general/commit/af64c9a432417b6694ae936fb5566fcfba763786,
```

I think it is enough to mention the c.g version that commit first appeared in:
```suggestion
    xfconf - generating facts was deprecated in community.general 3.0.0,
```